frog/imports/client/SingleActivity/ConfigPanel.js

```diff
@@ -21,9 +21,11 @@ import { style } from './style';
  */
 class ConfigPanel extends React.Component<
   {
+    name: string,
     activityType: ActivityPackageT,
     onSubmit: Function,
-    onReturn: Function
+    onReturn: Function,
+    data?: Object
```

Comment:
What is the format of data?

Reply:
This is the result of react-jsonschema-form for operators/activities, so Object is the most specific we can be.